src/iuse_actor.cpp

```diff
@@ -3223,14 +3236,26 @@ int heal_actor::finish_using( player &healer, player &patient, item &it, hp_part
     };
 
     if( patient.has_effect( effect_bleed, bp_healed ) ) {
-        if( x_in_y( bleed, 1.0f ) ) {
-            patient.remove_effect( effect_bleed, bp_healed );
-            heal_msg( m_good, _( "You stop the bleeding." ), _( "The bleeding is stopped." ) );
+        // small band-aids won't stop big arterial bleeding, but with tourniquet they just might
+        int pwr = 3 * get_stopbleed_level( healer );
+        pwr = patient.worn_with_flag( "TOURNIQUET", convert_bp( bp_healed ) ) ? pwr * 2 : pwr;
+        if( pwr > patient.get_effect_int( effect_bleed, bp_healed ) ) {
+            effect &wound = patient.get_effect( effect_bleed, bp_healed );
+            time_duration dur = wound.get_duration() - ( get_stopbleed_level( healer ) *
+                                wound.get_int_dur_factor() );
+            wound.set_duration( dur < 1_turns ? 0_turns : dur );
```

Comment:
If you're worried that `std::min` might not work with time durations, my compiler isn't complaining.
```suggestion
            wound.set_duration( std::min( 0_turns, dur ) );
```

Reply:
I think that should be `max`, not `min`.

Reply:
You're right, thanks.